docs/reference/migration/migrate_2_0.asciidoc

```diff
@@ -125,6 +125,12 @@ The `histogram` and the `date_histogram` aggregation now support a simplified `o
 `post_offset` rounding options. Instead of having to specify two separate offset shifts of the underlying buckets, the `offset` option
 moves the bucket boundaries in positive or negative direction depending on its argument.
 
+The `date_histogram` options for `pre_zone` and `post_zone` are replaced by the `time_zone` option. So far the `time_zone` option was
```

Comment:
I agree, `time_zone` basically replaces the old `pre_zone` and the explanation is too verbose here. However I would stress here that the rounding values returned by round(utcTime) which should be equivalent to keyForValue(roundKey(utcTime)) (and thus be used as the bucket keys) are always UTC now. Before you could shift them to any other time zone using `post_zone` which to avoid was one of the points of #9062. At least thats what I thought it was, but I could also be mistaken.